sql/mob_family_mods.sql

```diff
@@ -784,20 +775,25 @@ INSERT INTO `mob_family_mods` VALUES (252,51,2,1);  -- ROAM_TURNS: 2
 INSERT INTO `mob_family_mods` VALUES (252,52,30,1); -- ROAM_RATE: 30
 
 -- Wamoura
-INSERT INTO `mob_family_mods` VALUES (253,3,50,1); -- MP_BASE: 50
-INSERT INTO `mob_family_mods` VALUES (253,10,6,1); -- SUBLINK: 6
+INSERT INTO `mob_family_mods` VALUES (253,3,50,1);  -- MP_BASE: 50
+INSERT INTO `mob_family_mods` VALUES (253,10,6,1);  -- SUBLINK: 6
+INSERT INTO `mob_family_mods` VALUES (253,63,12,0); -- DEFP: 12
 
 -- Wamouracampa
 INSERT INTO `mob_family_mods` VALUES (254,10,6,1); -- SUBLINK: 6
 
 -- Wanderer
 INSERT INTO `mob_family_mods` VALUES (255,3,50,1); -- MP_BASE: 50
+INSERT INTO `mob_family_mods` VALUES (255,68,40,0); -- EVA: 40
```

Comment:
40 evasion is actually insane! lol
Where does this come from again?

Reply:
Jimmayus testing of Monster Stats